test/test_ops_gradients.py

```diff
@@ -30,6 +31,9 @@ class TestBwdGradients(TestGradients):
     # Tests that gradients are computed correctly
     @_gradcheck_ops(op_db + hop_db + custom_op_db)
     def test_fn_grad(self, device, dtype, op):
+        if platform.machine() == "s390x":
```

Comment:
Why not use IS_S390 here?

Reply:
I'll double check if this is possible and will update it if it's possible.